deps.bzl

```diff
@@ -4625,6 +4655,12 @@ def install_go_mod_dependencies(workspace_name = "buildbuddy"):
         sum = "h1:BVoBIqAf/2QdbFmSwAWnaIqDivZdOV0ZRwEm6jivLKw=",
         version = "v1.4.11",
     )
+    go_repository(
+        name = "com_github_tetratelabs_wazero",
+        importpath = "github.com/tetratelabs/wazero",
+        sum = "h1:IcZ56OuxrtaEz8UYNRHBrUa9bYeX9oVY93KspZZBf/I=",
+        version = "v1.9.0",
+    )
```

Comment:
Side note: im very impressed with rules_go+gazelle that wazero would __just work__ 😄